reviews/views.py

```diff
@@ -243,7 +243,7 @@ class ReviewAssignView(GroupRequiredMixin, AutoReviewMixin, generic.UpdateView):
 
     def get_success_url(self):
         committee = self.object.proposal.reviewing_committee.name
```

Comment:
Pretty sure this line can be removed

Same comment applies to the other modified ``get_success_url``

Reply:
You mean because the 'committee' is not referenced anywhere else? I was not sure what it is meant to do, so I left it alone...

Reply:
Yes; Previously it was used as a variable for the URL (to make sure you went to the right version of the page).

Now it's not needed anymore, thus should be removed